tests/cephadm/test_host_status_post_upgrade.py

```diff
@@ -0,0 +1,60 @@
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError, UnexpectedStateError
+from cli.ops.host import check_host_status
+from cli.utilities.utils import bring_node_offline
+from utility.log import Log
+
+log = Log(__name__)
+
```

Comment:
HI @sayaleeraut 

You can create method here for "check_host_status" as per your requirment

Thanks,
Mohit Bisht

Reply:
Done, added a local method to the test file itself.